pkgs/tools/networking/besttrace/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation rec {
+  pname = "besttrace";
+  version = "1.3.2";
+
+  src = fetchzip {
+    url = "https://cdn.ipip.net/17mon/besttrace4linux.zip";
```

Comment:
This URL does not seem to be immutable, any alternative avialable?

Reply:
This URL is application  official proviced, not other URL.

Reply:
This can cause troubles in the long run, but guess we have no choice now but to leave it as is.

Reply:
Can use other personal repo? like https://github.com/zhucaidan/BestTrace-Linux

Reply:
Or maybe https://archive.org

Reply:
it mainly depends on whether the license allows redistribution.

Reply:
It not license description, don't known whether allows redistribution.

Reply:
I will cancel this PR